css/main.css

```diff
@@ -0,0 +1,964 @@
+/*=== RESET TO INTUITIVE BOX LAYOUT ===*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+/*=== ICONS ===*/
+@font-face {
+  font-family: 'avengers_creative_icons';
+  src:url('/fonts/avengers_creative_icons.eot?nvpozf');
+  src:url('/fonts/avengers_creative_icons.eot?#iefixnvpozf') format('embedded-opentype'),
+    url('/fonts/avengers_creative_icons.ttf?nvpozf') format('truetype'),
+    url('/fonts/avengers_creative_icons.woff?nvpozf') format('woff'),
+    url('/fonts/avengers_creative_icons.svg?nvpozf#avengers_creative_icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'avengers_creative_icons';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+
+  /*=== BETTER FONT RENDERING ===*/
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+.icon-big {
+  font-size: 3em;
+  background-color: white;
+  color: black;
+  width: 100%;
+  border-radius: 50%;
+  padding: 1.25rem 1.25rem;;
+}
+
+.icon-cog:before {
+  content: "\e600";
+}
+
+.icon-lab:before {
+  content: "\e601";
+}
+
+.icon-brush:before {
+  content: "\e602";
+}
+
+.icon-pin:before {
+  content: "\e603";
+}
+
+.icon-mail:before {
+  content: "\e604";
+}
+
+.icon-mobile:before {
+  content: "\e605";
+}
+
+.s-icon-circle {
+  text-decoration: none;
+  color: #fd6632;
+  font-size: 3.25rem;
+  padding: 10%;
+  margin: auto;
+}
+
+.s-icon-circle:hover {
+  color: #0b9ce0;
+}
+
+.icon-circle-right {
+  order: 3;
+}
+
+.icon-circle-right:before {
+  content: "\e606";
+}
+
+.icon-circle-left {
+  order: 1;
+}
+
+.icon-circle-left:before {
+  content: "\e607";
+}
+
+.icon-social {
+  text-decoration: none;
+  font-size: 2rem;
+  padding: 1%;
+}
+
+.icon-google-plus2 {
```

Comment:
Is there a reason this cannot be `.icon-google-plus`? It looks like you custom-built your font to load just the characters you wanted (which is awesome), but this class name is a little funny since none of the others end in a number.